source/patterns/@aws-solutions-constructs/aws-lambda-elasticsearch-kibana/test/lambda-elasticsearch-kibana.test.ts

```diff
@@ -146,4 +131,333 @@ test('check override cognito domain name with provided cognito domain name', ()
   expect(stack).toHaveResource('AWS::Cognito::UserPoolDomain', {
     Domain: 'test-cognito-domain'
   });
-});
\ No newline at end of file
+});
+
+test('Check for 3 subnets for environment specified stack', () => {
+  const stack = new cdk.Stack(undefined, undefined, {
+    env: { account: "123456789012", region: 'us-east-1' },
+  });
+
+  const lambdaProps: lambda.FunctionProps = {
+    code: lambda.Code.fromAsset(`${__dirname}/lambda`),
+    runtime: lambda.Runtime.NODEJS_12_X,
+    handler: 'index.handler',
+  };
+
+  new LambdaToElasticSearchAndKibana(stack, 'test-lambda-elasticsearch-kibana', {
+    lambdaFunctionProps: lambdaProps,
+    domainName: "test",
+    deployVpc: true,
+  });
+
+  expect(stack).toCountResources("AWS::EC2::VPC", 1);
+  expect(stack).toCountResources("AWS::EC2::Subnet", 3);
+});
```

Comment:
It's not clear to me from index.ts that the ES domain is getting put in a VPC, probably because of the way VPC is passed to the ES domain. Can we add an expect here that checks if the ES domain is in the VPC, rather than just checking the existence of the VPC?